lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2476,7 +2476,7 @@ static function ( $pseudo_selector ) use ( $selector ) {
 	 */
 	public function get_root_layout_rules( $selector, $block_metadata ) {
 		$css              = '';
-		$settings         = _wp_array_get( $this->theme_json, array( 'settings' ) );
+		$settings         = $this->theme_json['settings'] ?? array();
```

Comment:
Previously this didn't have a fallback value. It doesn't appear to cause any issue making it `array()` here, though as far as I can tell!

Reply:
Yeah, this did not have a default value, which means it would previously return `null`. 
However, a few lines below we're treating `$settings` as an array so for the sake of consistency I made it an array. We _could_ make it `null` instead and nothing would change since the code treating `$settings` is inside an `isset()` check, this just seemed more appropriate 👍